Makefile

```diff
@@ -24,7 +24,8 @@ build-and-push-airlock-processor: build-airlock-processor push-airlock-processor
 deploy-shared-services: firewall-install
 	. ${MAKEFILE_DIR}/devops/scripts/load_env.sh ./templates/core/.env \
 	&& if [ "$${DEPLOY_GITEA}" == "true" ]; then $(MAKE) gitea-install; fi \
-	&& if [ "$${DEPLOY_NEXUS}" == "true" ]; then $(MAKE) nexus-install; fi
+	&& if [ "$${DEPLOY_NEXUS}" == "true" ]; then $(MAKE) nexus-install; fi \
+	&& if [ "$${DEPLOY_AIRLOCK_NOTIFIER}" == "true" ]; then $(MAKE) airlock-notifier-install; fi
```

Comment:
We're moving away from installing shared services from within the makefile. Can you revert all changes related to this (here and in the other files as well)?
Documentation should point users to register the bundle and use an API (or UI) to ask the TRE to install it.